src/MagnumPlugins/TinyGltfImporter/Test/required-extensions-unsupported.gltf

```diff
@@ -0,0 +1,9 @@
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsRequired": [
+        "KHR_materials_pbrSpecularGlossiness",
+        "EXT_lights_image_based"
+    ]
```

Comment:
I'd say the (non-invalid) test files should definitely pass the validator :eyes: I'll fix that.

As for checking that in the importer... I wouldn't bother. We *could* error out, but once the file is corrected, the import behaviour is the same. It kind of depends on how spec-conformant you want the importer to be and how much validation you want to perform. There are a few cases already where the importer allows 'invalid' (per the spec) files because it really makes no sense to enforce those.

Reply:
Agree. Yep, it already allows scene-less files and other things not allowed by spec, so why not this as well.